### DISCUSSION TITLE
What type of Partition to use

### DISCUSSION THREAD

**Discussion by joaocasarin** [ventoy/Ventoy#2503] [Q&A]
Basically I just bought a 128GB pendrive to have a few ISOs in it. They will vary from Windows 10 basic version like Home 64bits to Pro 64bits. But at the same time I want to have Ubuntu, Debian or whatever Linux distro I want in the same pendrive to install on my laptops.

What should I choose, MBR or GPT?

**Comment by mrgnex** [comment]
GPT for UEFI and MBR for legacy/BIOS
